src/components/shared/Extensions/Accordion/partials/SpecialHourInput.tsx

```diff
@@ -3,31 +3,35 @@ import React, { FC } from 'react';
 import SpecialInput from '~common/Extensions/SpecialInput';
 import FormError from '~shared/Extensions/FormError';
 import { SpecialInputProps } from '../types';
+import ConnectForm from './ConnectForm';
 
 const displayName = 'Extensions.SpecialHourInput';
 
 const SpecialHourInput: FC<SpecialInputProps> = ({
   defaultValue,
   maxValue,
-  register,
   errors,
   name = '',
 }) => (
-  <div className="text-right">
-    <div className="flex justify-end flex-col w-[8.75rem]">
-      <SpecialInput
-        defaultValue={defaultValue as number}
-        register={register}
-        isError={!!errors?.[name]?.message}
-        name={name}
-        type="hours"
-        min={1}
-        max={maxValue}
-        placeholder="1"
-      />
-      {errors?.[name] && <FormError>{errors?.[name]?.message}</FormError>}
-    </div>
-  </div>
+  <ConnectForm>
+    {({ register }) => (
+      <div className="text-right">
+        <div className="flex justify-end flex-col w-[8.75rem]">
```

Comment:
It is better to use `max-width` instead of `width` because it's more responsive when you resize your browser to small breakpoints `w-full max-w-[8.75rem]`